service/src/test/java/bio/terra/catalog/rawls/RawlsServiceTest.java

```diff
@@ -0,0 +1,62 @@
+package bio.terra.catalog.rawls;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+import bio.terra.catalog.model.SystemStatusSystems;
+import bio.terra.rawls.client.ApiException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@ConfigurationPropertiesScan("bio.terra.catalog")
+@ContextConfiguration(classes = {RawlsService.class})
+class RawlsServiceTest {
+  @Autowired private RawlsService rawlsServiceReal;
+
+  private RawlsService rawlsService;
+
+  @BeforeEach
+  void beforeEach() {
+    rawlsService = spy(rawlsServiceReal);
+  }
+
+  @Test
+  void status() throws Exception {
+    SystemStatusSystems status = new SystemStatusSystems().ok(true);
+    when(rawlsService.status()).thenReturn(status);
```

Comment:
It's not a big deal but to make this test structured like the other service tests (`SamServiceTest` and `DatarepoServiceTest`), it would be more consistent to mock the method that creates the API, instead of a higher level method. In this case you'd mock a new method like `statusApi()`.

The main benefit of writing a test this way is you get coverage of the code that immediately surrounds the service's API layer. There's not much code for `status` but there will be more for other Rawls operations.

Reply:
The problem is that since `StatusApi().systemStatus()` returns `void`, I get an error if I try to mock this call. I'm not sure if there's a way to solve this.